src/garage/sampler/off_policy_vectorized_sampler.py

```diff
@@ -59,7 +59,7 @@ def shutdown_worker(self):
         """Terminate workers if necessary."""
         self.vec_env.close()
 
-    def obtain_samples(self, itr, batch_size):
+    def obtain_samples(self, itr, batch_size, add_transitions_to_buffer=True):
```

Comment:
So if I set this to False, the transitions don't get added to the replay buffer? Why is that useful?